src/bin/tikv-ctl.rs

```diff
@@ -1100,6 +1143,23 @@ fn main() {
     }
 }
 
+fn get_module_type(module: &str) -> MODULE {
+    let module_type = match module {
+        "rocksdb" => MODULE::ROCKSDB,
```

Comment:
I don't think `rocksdb` is a good name. Both `KV` and `RAFT` are rocksdb in fact. So I think you can use `KVDB`.

Reply:
`rocksdb` is a old name when we mix `kv` and `raft log` in one `rocksdb`. @zhangjinpeng1987 any better name or keep it?

Reply:
I prefer kvdb and raftdb too.